src/emu/rendersw.hxx

```diff
@@ -29,27 +29,27 @@ private:
 	};
 
 	// internal helpers
-	static inline bool is_opaque(float alpha) { return (alpha >= (_NoDestRead ? 0.5f : 1.0f)); }
-	static inline bool is_transparent(float alpha) { return (alpha < (_NoDestRead ? 0.5f : 0.0001f)); }
-	static inline rgb_t apply_intensity(int intensity, rgb_t color) { return color.scale8(intensity); }
-	static inline float round_nearest(float f) { return floor(f + 0.5f); }
+	static constexpr bool is_opaque(float alpha) { return (alpha >= (_NoDestRead ? 0.5f : 1.0f)); }
+	static constexpr bool is_transparent(float alpha) { return (alpha < (_NoDestRead ? 0.5f : 0.0001f)); }
+	static constexpr rgb_t apply_intensity(int intensity, rgb_t color) { return color.scale8(intensity); }
```

Comment:
Will all compilers accept this?  `rgb_t::scale8` modifies a local, which is something MSVC used to reject.